app/Http/Controllers/Admin/PostController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Admin\AdminController;
+use App\Models\Post;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Spatie\Valuestore\Valuestore;
+
+class PostController extends AdminController
+{
+    private $paginationNum = 0;
+
+    public function __construct()
+    {
+        $this->audiences = Post::getAudiences();
+        $settings = Valuestore::make(storage_path('app/settings.json'));
+        $this->paginationNum = $settings->get('post_pagination', 0);
+    }
+
+    public function index($id)
+    {
+        $postList = User::find($id)->posts()->newestPosts()->paginate($this->paginationNum);
```

Comment:
Cái `$id` này là của user là của post?

Tham khảo guide về nested resource như a đã gửi để viết lại toàn bộ phần này đi nhé